task_spec/spec/acceptance/init_spec.rb

```diff
@@ -57,6 +59,15 @@ def target_platform
       expect(res['result']['source']).to be
     end
 
+    service = if target_platform =~ /win/
+                run_command('c:/"program files"/"puppet labs"/puppet/bin/puppet resource service puppet', 'target')
+              else
+                run_command('/opt/puppetlabs/bin/puppet resource service puppet', 'target')
+              end
+    output = service[0]['result']['stdout']
+    expect(output).to include("ensure => 'stopped'")
+      .or include("ensure => 'absent'")
```

Comment:
Now that we are not stopping `pxp-agent` will `puppet` ever be `absent`?

Reply:
Seems like no!